scss/_settings_colors.scss

```diff
@@ -61,7 +61,7 @@ $states: (
 //
 // Background colors:
 // --background           - default background color
-// --background-alt       - alternative version of background color, to be used to diffrenciate given surface
+// --background-alt       - alternative version of background color, to be used to differentiate given surface
 // --background-highlight - background used on highlighted elements (for example selected elements in the navigation)
```

Comment:
Comment needs to be updated, we don't have `--background-highlight`, instead both `--background-hover` and `--background-active` should be documented in the comment

Reply:
updated